OpenOversight/app/templates/list_officer.html

```diff
@@ -163,7 +163,7 @@ <h3 class="panel-title accordion-toggle">Unique ID</h3>
       {% for officer in officers.items %}
         <li class="list-group-item">
             <div class="row">
-              <div class="col-md-6 col-xs-12">
+              <div class="col-md-6 col-xs-12 center-text">
```

Comment:
This centers the images in the list:
<img width="422" alt="Screenshot 2023-07-10 at 1 54 35 PM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/f32d1cb8-fa80-4dcc-80fd-4e56f1eaad14">